src/extension/mapml/src/main/java/org/geoserver/mapml/MapMLFeatureUtil.java

```diff
@@ -555,4 +828,82 @@ private static boolean getBoleanFormatOption(Request request, String key) {
         if (formatOptions != null && Boolean.valueOf((String) formatOptions.get(key))) return true;
         return false;
     }
+
+    /** Convenience class to hold the information needed to simplify a feature collection */
+    public static class FeatureCollectionInfoSimplifier {
```

Comment:
The name is misleading... it can contain feature collections or coverages, it does indeed have a simplifier but performs no action (basically just a DTO). Maybe "LayerSimplifierContext" instead?

Reply:
renamed